helm/charts/paradedb/templates/auth.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.auth }}
+{{- if .Values.auth.appUserCredentials }}
+apiVersion: v1
+data:
+  password: {{ .Values.auth.appUserCredentials.password }}
+  username: {{ .Values.auth.appUserCredentials.username | b64enc }}
+kind: Secret
+metadata:
+  name: {{ .Values.auth.appUserCredentials.name }}
+  labels: 
+{{include "cluster.labels" . | indent 4 }}
+type: kubernetes.io/basic-auth
+{{- end }}
+---
```

Comment:
what do these delimiters do? They're not in the cluster.yaml

Reply:
They split different documents or resources in YAML. This way you can define multiple resources on a single file.